tests/scripts/task_python_integration.sh

```diff
@@ -64,7 +64,7 @@ run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-integration tests/python/int
 run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-contrib tests/python/contrib --ignore=tests/python/contrib/test_ethosu --ignore=tests/python/contrib/test_cmsisnn
 # forked is needed because the global registry gets contaminated
 TVM_TEST_TARGETS="${TVM_RELAY_TEST_TARGETS:-llvm;cuda}" \
-    run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-relay tests/python/relay
+    run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-relay tests/python/relay --ignore=tests/python/relay/aot
```

Comment:
nit: we probably don't need to ignore these on ci_cpu as the irrelevant tests will be skipped

Reply:
Yes I was thinking about it, but there are also few hundred tests in that folder that won't be skipped on neither `ci_cpu` nor `ci_cortexm` and are not really dependent on anything in these environments, so I don't see a reason to duplicate these tests.

Reply:
good point, let's leave it :)